ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -194,10 +195,17 @@
     {/each}
 
     <li class="item" data-cy="add-org-button">
-      <AddOrgButton on:click={() => dispatch('createorg')} />
-      <div class="tooltip">
-        <Title>Add org</Title>
-      </div>
+      {#if registerOrgPermission}
```

Comment:
```suggestion
    <AddOrgButton 
      on:click={() => registerOrgPermission && dispatch('createorg')} 
      disabled={!registerOrgPermission}
     />
    <div class="tooltip">
      <Title>
        {registerOrgPermission ? 'Register your handle to create an org' : 'Add org}
      </Title>
    </div>
```

Reply:
I find it easier to read to keep the `if` statements outside of the component. In general I think avoiding repetition is good, but here it doesn't add too much duplication.